### PR TITLE
chore: address some of no-unused-vars

### PR DESCRIPTION
**Pull request by sobolk**
#### Description of changes

This knocks off 77 lint warnings.

Changes are mostly unused imports and unused settings in e2e test helpers.

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [ ] [Pull request labels](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#labels) are added

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-e2e-core/src/categories/api.ts [hunk 1]
```diff
@@ -689,7 +689,7 @@ function setupAuthType(authType: string, chain: any, settings?: any) {
       setupCognitoUserPool(chain);
       break;
     case 'IAM':
-      setupIAM(chain);
+      // no-op
```
Comment (sobolk) [inline packages/amplify-e2e-core/src/categories/api.ts:692]: image

File: packages/amplify-e2e-core/src/categories/storage.ts [hunk 2]
```diff
@@ -314,22 +314,22 @@ export function addDynamoDBWithGSIWithSettings(projectDir: string, settings: Add
 
     chain.wait('Provide a friendly name').sendLine(settings.resourceName).wait('Provide table name').sendLine(settings.tableName);
 
-    addColumn('pk', 'string');
```
Comment (sobolk) [inline packages/amplify-e2e-core/src/categories/storage.ts:317]: image

**Review by edwardfoyle — DISMISSED** [review]

File: packages/amplify-provider-awscloudformation/src/resource-package/resource-export.ts [hunk 3]
```diff
@@ -431,7 +431,7 @@ export class ResourceExport extends ResourcePackager {
         // remove url parameters will set it in the construct
         // remove template URL the stack, CDK will update the URL
         if (deleteParameters) {
-          const { Parameters, TemplateURL, ...others } = template.Resources[resourceKey].Properties;
+          const { Parameters, ...others } = template.Resources[resourceKey].Properties;
```
Comment (jhockett) [inline packages/amplify-provider-awscloudformation/src/resource-package/resource-export.ts:434]: I'm not sure if it matters, but I think this changes the the behavior on line 437. By pulling out `TemplateURL` into it's own variable, it's being excluded from `resource.Properties`. Same with change on line 445.

Comment (sobolk) [reply packages/amplify-provider-awscloudformation/src/resource-package/resource-export.ts:434]: I'll descope it then for now.

**Review by edwardfoyle — APPROVED** [review]

**Review by jhockett — APPROVED** [review]

**Pull request merged**